discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -112,6 +123,7 @@ fn spawn_headlines_request(
     client: Arc<Client>,
     market: Market,
     page_size: usize,
+    excluded_sources: Vec<String>,
```

Comment:
🟡  we could wrap the vector in another `Arc` to avoid cloning the vector everytime we make a request, ie `BreakingNews` would have the field `excluded_sources: Option<Arc<RwLock<Arc<Vec<String>>>>>`. similarly for `PersonalizedNews`, wdyt?
```suggestion
    excluded_sources: Arc<Vec<String>>,
```

Reply:
Yes I wanted to fix the unnecessary cloning in a follow up refactoring (which also reduces the number of locks and arcs) through until then but this works, too.